cmd/image-builder/upload_test.go

```diff
@@ -65,7 +65,7 @@ func TestUploadWithAWSMock(t *testing.T) {
 		{"fake-disk.img", "", arch.Current().String(), fmt.Sprintf("WARNING: no upload architecture specified, using %q (use --arch to override)\n", arch.Current().String())},
 	} {
 		fakeImageFilePath := filepath.Join(t.TempDir(), tc.fakeDiskName)
-		err := os.WriteFile(fakeImageFilePath, []byte(fakeDiskContent), 0644)
+		err := os.WriteFile(fakeImageFilePath, []byte(fakeDiskContent), 0o644)
```

Comment:
This has no effect, just for the record.

Reply:
It was a copy & paste from one of the other tests lol

Reply:
Ah sorry it was my editor. Let me revert that.

Reply:
Ya no that's fine, I personally just prefer to keep changes to the minimal and not change unrelated code :)